src/app/components/navbar/NavBar.test.js

```diff
@@ -46,7 +47,7 @@ describe("NavBar", () => {
         it("should render navigation with links", () => {
             const component = shallow(<NavBar {...defaultProps} user={authenticatedUser} />);
             const nav = component.find(Link);
-
+            
```

Comment:
trailing whitespaces, worth before opening PR run lint so your code is formatted and those errors removed